address

```diff
@@ -1,19 +0,0 @@
-{
```

Comment:
In the case where the file already existed, yes.  It is odd.  If it were a new symlink only, you would only see the addition indicating the location where you made your other comment.